cosmo_tester/framework/test_hosts.py

```diff
@@ -885,11 +886,20 @@ class Cloudify4_6Manager(_OldStatusFormat, _CloudifyManager):
     branch_name = '4.6'
 
 
+class Cloudify5_0_5Manager(_CloudifyManager):
+    branch_name = '5.0.5'
+
+
 class Cloudify4_6Manager_ClusterJoined(Cloudify4_6Manager):
     def verify_services_are_running(self):
         pass
 
 
+class Cloudify5_0_5Manager_ClusterJoined(Cloudify5_0_5Manager):
+    def verify_services_are_running(self):
```

Comment:
Why is this needed?

Reply:
Not sure. I figured out that it's needed for cluster to bypass the fact that the manager is not an AIO and doesn't have all services locally, but maybe I'm wrong about it. The framework is not very transparent